### DISCUSSION TITLE
如何配置锚点?

### DISCUSSION THREAD

**Discussion by vaynevayne** [umijs/dumi#1662] [Q&A]
我的理解: group.title 表示锚点, 一个md文件只能写一个title, 所以渲染多个锚点, 需要多个md, 

但是我添加另一个md文件, 并不会生效,应该怎么设置锚点呢? 

```
// Table/index.md

---
group:
  title: index锚点
---


这个不会生效
// Table/a.md
---
group:
  title: a锚点
---
```

**Comment by PeachScript** [comment]
`group.title` 并非锚点，只是侧边菜单的分组名称，当同一个导航下的多个 md 都拥有同样的 `group.title` 时，它们就会归类到同一分组展示

**Reply by vaynevayne** [reply]
哦哦,这样啊
